src/markets.json

```diff
@@ -2921,5 +2921,11 @@
     "address": "3PB2WZB7kiT9fi8NyiSN3gorkC9Ro5pRCPzr9VqWgJSZ",
     "programId": "BookWizfXJLgda24JdN1drgWzqw3ta3rU1vGotFjpNTH",
     "deprecated": false
+  },
+  {
+    "name": "B4C3/USDC",
+    "address": "FoMNYEhKUMfcRdU8TswjoABD5CroEriTat4EhFayGpwK",
```

Comment:
This is not `marketId`, this is `tokenId`. 
Please apply correct marketId in order to list this market.

Reply:
Updated with marketId.